defaults/main.yml

```diff
@@ -13,9 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Can be used to completely disable the role's handler actions
-haproxy_management: "{{ haproxy_hosts | default([]) | length > 0 }}"
```

Comment:
Let's leave this in and default it to yes so there is still a way for someone to completely disable the role actions even if haproxy is present.

Reply:
Actually I wonder if it might work to simply remove the jinja brackets from this var default and set it to:
```
haproxy_management: 'haproxy_hosts | default([]) | length > 0'
```

with no other changes

Reply:
You'd still need that jinja2 to be evaluated, though.